// Copyright 2023 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//    https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package guideline.vision;

// Options for the OccupancyMapCalculator
message OccupancyMapCalculatorOptions {
  int32 width_meters = 1;
  int32 depth_meters = 2;
  float bottom_meters = 3;
  float top_meters = 4;
  float point_confidence_threshold = 5;
  float occupancy_threshold = 6;
}
